obelix/idefix/devAuth/gclient.go

```diff
@@ -0,0 +1,48 @@
+package devAuth
+
+import (
+	"context"
+	"fmt"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"net/http"
+	"os"
+)
+
+var (
+	CLIENT_ID     = ""
+	CLIENT_SECRET = ""
+	REDIRECT_URL  = ""
+)
+
+func init() {
+	CLIENT_ID = os.Getenv("GOOGLE_CLIENT_ID")
+	CLIENT_SECRET = os.Getenv("GOOGLE_CLIENT_SECRET")
+	REDIRECT_URL = os.Getenv("GOOGLE_REDIRECT_URL")
```

Comment:
return ""

Reply:
And then it will make your program fails.
If this will not be replaced by `credentialAPI`, you should correctly handle that kind of error.
If not, your program will run but fail after a call to google API